.github/workflows/integrate.yaml

```diff
@@ -43,6 +43,13 @@ jobs:
     - name: Run unit tests
       run: tox -e unit
 
+  terraform-checks:
+    name: Terraform
+    uses: canonical/charmed-kubeflow-workflows/.github/workflows/terraform-checks.yaml@main
+    with:
+      charm-path: .
+      channel: latest/edge
```

Comment:
We should omit this according to discussion https://github.com/canonical/training-operator/pull/200#discussion_r1772484185